packages/js/components/src/form-section/stories/index.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { createElement } from '@wordpress/element';
+import { Card, CardBody, TextControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { FormSection } from '../';
+
+export const Basic: React.FC = () => {
+	return (
+		<FormSection
+			title="My form section"
+			description="Some text to describe what this section covers"
+		>
+			<Card>
```

Comment:
@louwie17 I removed the `Card` styles and rely directly on the `Card` component since some sections contain multiple cards within the content area, but let me know if you have any issue with this.

Reply:
Thanks, I always forget about the `Card` component, relying on that makes much more sense.